website/snippets/core-versions-table.md

```diff
@@ -6,10 +6,10 @@
 | [**v1.4**](/guides/migration/versions/upgrading-to-v1.4)   | Jan 25, 2023    | Critical      | Jan 25, 2024            | 
 | [**v1.3**](/guides/migration/versions/upgrading-to-v1.3)   | Oct 12, 2022    | Critical      | Oct 12, 2023            | 
 | [**v1.2**](/guides/migration/versions/upgrading-to-v1.2)   | Jul 26, 2022    | Critical      | Jul 26, 2023            |
-| [**v1.1**](/guides/migration/versions/upgrading-to-v1.1) ⚠️ | Apr 28, 2022    | End of Life ⚠️ | Apr 28, 2023            | 
-| [**v1.0**](/guides/migration/versions/upgrading-to-v1.0) ⚠️ | Dec 3, 2021     | End of Life ⚠️ | Dec 3, 2022 ⚠️           | 
-|  **v0.X** ⚠️                                                | (Various dates) | End of Life ⚠️ | Deprecated ⚠️            | 
-
+| [**v1.1**](/guides/migration/versions/upgrading-to-v1.1) ⚠️ | Apr 28, 2022    | End of Life* ⚠️ | Apr 28, 2023            | 
+| [**v1.0**](/guides/migration/versions/upgrading-to-v1.0) ⚠️ | Dec 3, 2021     | End of Life* ⚠️ | Dec 3, 2022 ⚠️           | 
+|  **v0.X** ⛔️                                                | (Various dates) | Deprecated ⛔️ | Deprecated ⛔️            | 
+_*All EOL dbt Core v1.0 and higher are available in dbt Cloud until further notice._
```

Comment:
How about something like:
```suggestion
_*All versions of dbt Core since v1.0 are available in dbt Cloud until further notice. Versions that are EOL do not receive any fixes. For best support, we recommend upgrading to a version released within the past 12 months._
```